docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM rhub/ubuntu
```

Comment:
do we need rhub here? Would it not perhaps be preferable to use the official ubuntu image? https://hub.docker.com/_/ubuntu